ci/common-env

```diff
@@ -0,0 +1,5 @@
+export OCI_TENANCY_ID=ocid1.tenancy.oc1..aaaaaaaanorijchxlkb4zbectmyxugdvqjbjyjg4g7664tdnwwyj3eqsxqha
+export OCI_REGION=us-ashburn-1
+# The `benchmark-tests` compartment
+export OCI_COMPARTMENT_ID="ocid1.compartment.oc1..aaaaaaaaqn6nchyedtj26kt33244nm5bfi4qejjuknup4353n6y7a5ib24cq"
+export SUPPRESS_LABEL_WARNING=True
```

Comment:
These are safe to expose publicly and commit right?

Reply:
Yeah, they're just IDs for the tenancy. Not sure if that should be considered secret